frontend_tests/zjsunit/index.js

```diff
@@ -61,6 +62,7 @@ global.stub_i18n = require('./i18n.js');
 const handlebars = require('./handlebars.js');
 global.make_handlebars = handlebars.make_handlebars;
 global.stub_templates = handlebars.stub_templates;
+global.rewiremock = rewiremock;
```

Comment:
Let’s not add more globals. Just `require()` it from the test files where it’s used.